src/kps/gui/windows/form/dialogs/LocationDialog.java

```diff
@@ -6,27 +6,34 @@
 
 import java.awt.*;
 
+/**
+ * Dialog for updating and creating new locations
+ */
 public class LocationDialog extends FormDialog {
     public static final String NAME = "Name";
 
+    /**
+     * Main constructor for the locations dialog
+     * @param owner the frames owner
+     * @param title the frame title
+     * @param modal whether the frame should disable background windows
+     * @param simulation the simulation which the location should be updated or added t
+     */
     public LocationDialog(Frame owner, String title, boolean modal, Simulation simulation) {
         super(owner, title, modal, simulation);
         buildDialog();
         setVisible(true);
     }
 
-    @Override
-    protected void initializeForm(FormBuilder builder) {
+    @Override protected void initializeForm(FormBuilder builder) {
```

Comment:
I think this could do with a simple comment, something like initialises a form with a location name would do.